packages/components/src/components/text/text.jsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import React from 'react';
+
+const Text = ({ children, size, color, align, weight, lineHeight, as }) => {
```

Comment:
Is it possible to provide sensible defaults for this? e.g. what happens if `weight` is undefined? 
What would be the font weight? 

Is it possible to explain the defaults in form of  comment or storybook?

Reply:
Mamali, I wanted to do that, but I think it is not needed since, CSS sets the default values for that, unless we want to set the defaults on our own. For example, default size is 16px. Is it needed to define them?

Reply:
Good point, However we need to make sure the output is optimized, it is unclear to me that what the output would be? I mean, does it render `style="--text-size: undefined"` or it does **not** render that variable at all? 
How about the other props, tho? 
`<Text>My text</Text>` is this the proper usage? If that is so, does it render a paragraph?

Reply:
Yes storybook would be useful 👍